project-base/app/tests/FrontendApiBundle/Functional/Cart/AuthenticatedCartModificationsResultTest.php

```diff
@@ -213,92 +213,6 @@ public function testCartItemWithModifiedPriceIsReported(): void
         self::assertEmpty($itemModifications['cartItemsWithChangedQuantity']);
     }
 
-    public function testCartItemWithChangedQuantityIsReported(): void
-    {
-        $productQuantity = 2;
-        $this->addTestingProductToNewCart($productQuantity);
-
-        $this->setOneItemLeftOnStockForTestingProduct();
-
-        $getCartQuery = '{
-            cart {
-                modifications {
-                    itemModifications {
-                        cartItemsWithChangedQuantity {
-                            uuid
-                            product {
-                                uuid
-                            }
-                        }
-                        noLongerAvailableCartItemsDueToQuantity {
-                            uuid
-                        }
-                        noLongerListableCartItems {
-                            uuid
-                        }
-                        cartItemsWithModifiedPrice {
-                            uuid
-                        }
-                    }
-                }
-            }
-        }';
-
-        $response = $this->getResponseContentForQuery($getCartQuery);
-        $data = $this->getResponseDataForGraphQlType($response, 'cart');
-        $itemModifications = $data['modifications']['itemModifications'];
-
-        self::assertNotEmpty($itemModifications['cartItemsWithChangedQuantity']);
-        self::assertEquals($this->testingProduct->getUuid(), $itemModifications['cartItemsWithChangedQuantity'][0]['product']['uuid']);
-
-        self::assertEmpty($itemModifications['noLongerListableCartItems']);
-        self::assertEmpty($itemModifications['noLongerAvailableCartItemsDueToQuantity']);
```

Comment:
all the occurences of "noLongerAvailableCartItemsDueToQuantity" in the codebase should be removed as well, right?

Reply:
Indeed it is. Removed it everywhere.